src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs

```diff
@@ -181,7 +181,7 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
             => compilation.GetTypeByMetadataName(typeof(ObsoleteAttribute).FullName);
 
         public static INamedTypeSymbol? SystemCompositionImportingConstructorAttribute(this Compilation compilation)
-            => compilation.GetTypeByMetadataName(typeof(System.Composition.ImportingConstructorAttribute).FullName);
+            => compilation.GetTypeByMetadataName("System.Composition.ImportingConstructorAttribute");
```

Comment:
😕 Why did this one change?

Reply:
I recall doing it because I got an error in code style layer. I can revert it for this PR, and if required in https://github.com/dotnet/roslyn/pull/41363, I will add it back.

Reply:
Yep, this gives compiler error in CodeStyle layer that "Composition" is not available in "System" namespace. Retaining this change.